xwiki-platform-core/xwiki-platform-release/xwiki-platform-release-ui/src/main/resources/Release/WebHome.xml

```diff
@@ -63,19 +59,20 @@
 {{/html}}
 
 = Existing Releases =
-#set($columns = ["version", "releaseManagers", "releaseDate", "releaseNotes"])
-#set($columnsProperties = {
-    "version" : { "type" : "text" , "link" : "view"},
-    "releaseManagers" : { "html" : true },
-    "releaseDate" : { "type" : "text" }
-})
-#set($options = { 
-   "className":"ReleaseCode.ReleaseClass",
-   "selectedColumn" : "releaseDate",
-   "defaultOrder" : "desc",
-   "rowCount": 15,
-   "translationPrefix":"release.livetable."
+
+#set($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+       { 'id': 'version', 'displayer': 'link' },
+       { 'id': 'releaseDate', 'name': $services.localization.render('release.livetable.releaseDate') }
```

Comment:
name needed to keep the label as `Release Date` instead of `Release Date` (the field name).

Reply:
shouldn't that info be added in the file instead of in a comment in the PR? :)

PS: I don't understand you comment (both value seem to the same).

Reply:
@manuelleduc from your comment I deduce that the field pretty name takes precedence over the specified translation prefix, which is not right. It should be the other way around: if the translation prefix is specified and the corresponding key has a value then it should be used, otherwise it should fallback on the field (translated) pretty name. This needs to be fixed in ``livetable.DefaultLiveDataConfigurationResolver``. It's not critical and can be fixed later. So please remove the ``releaseDate`` descriptor. Would be good to report an issue on the live data for the property name fallback issue.

Reply:
Issue created https://jira.xwiki.org/browse/XWIKI-18628 thanks.